charts/vcluster-rancher-extension-ui/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: vcluster-rancher-extension-ui
+description: A Helm chart for the vCluster Rancher Extension UI
+type: application
+# This version will be dynamically updated during the CI process with the git tag
+version: 0.0.3
+appVersion: 0.0.3
```

Comment:
Fine for now, but ideally this sort of thing can be templated/automated and pull directly from the tag.

Reply:
It is already pulled from the tag and changed in ci directly here:
![image](https://github.com/user-attachments/assets/90eda9c7-366f-4d95-9ac6-c829106ae6e0)